core.d.ts

```diff
@@ -511,7 +516,7 @@ export declare class FileTypeParser {
 	/**
 	Works the same way as {@link fileTypeFromStream}, additionally taking into account custom detectors (if any were provided to the constructor).
 	*/
-	fromStream(stream: ReadableStream): Promise<FileTypeResult | undefined>;
+	fromStream(stream: NodeReadableStream): Promise<FileTypeResult | undefined>;
```

Comment:
Shouldn't this and `toDetectionStream` support web streams too?

Reply:
Yeah, should probably be done as well.